tooling/nargo_fmt/tests/input/fn.nr

```diff
@@ -19,28 +19,45 @@ fn main(
 
 fn main<T>() where T: Eq {}
 
-fn main(tape: [Field; TAPE_LEN], initial_registers: [Field; REGISTER_COUNT], initial_memory: [Field; MEM_COUNT], initial_program_counter: Field, initial_call_stack: [Field; MAX_CALL_STACK], initial_call_stack_pointer: u64) -> pub ExecutionResult {}
-
-fn apply_binary_field_op<N>(lhs: RegisterIndex, rhs: RegisterIndex, result: RegisterIndex, op: u8, registers: &mut Registers<N>) -> bool {}
-
-fn main() -> distinct pub [Field;2] {}
+fn main(
```

Comment:
Why did this change? Is it not input?

Reply:
As `distinct` causes an error having it here prevents the test from running properly.

Reply:
I've reverted the other changes caused by automatic formatting when saving the file.